frontend/src/static/css/styles.scss

```
@@ -1196,10 +1196,10 @@ input#authenticator_secret {
 	padding: 10px 12px !important;
 	font-size: 12px !important;
 	line-height: 1.5 !important;
-	// color: #e3e3e3 !important;
-	color: rgba(#fff, 1) !important;
-	background-color: #595959 !important;
-	// background-color:#909090 !important;
+	// color: var(--cinemata-pacific-deep-50) !important;
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)